docs/start/framework/routing.md

```diff
@@ -50,6 +50,18 @@ export default [
 ```
 
 If you prefer to define your routes via file naming conventions rather than configuration, the `@react-router/fs-routes` package provides a [file system routing convention.][file-route-conventions]
+These approaches can also be combined, as shown below:
```

Comment:
```suggestion
If you prefer to define your routes via file naming conventions rather than configuration, the `@react-router/fs-routes` package provides a [file system routing convention][file-route-conventions]. Different routing conventions can even be combined:
```